pkg/cloudprovider/aws/fleet/capacity.go

```diff
@@ -71,8 +71,8 @@ func (c *Capacity) Create(ctx context.Context, constraints *cloudprovider.Constr
 	}
 	nodePackings := []cloudprovider.Packing{}
 	for instanceID, node := range nodes {
-		node.Labels = constraints.Labels
-		node.Spec.Taints = constraints.Taints
+		node.Labels = constraints.Constraints.Labels
```

Comment:
Is there some way to fix the repetition (constraints constraints) here?

Reply:
I might rename one of these concepts.

Reply:
Since it's an embedded struct, I can simply remove the second constraints without any other code changes. Thanks